src/main/asciidoc/_chapters/ops_mgt.adoc

```diff
@@ -3479,6 +3478,27 @@ To enable ACL, add the following to your hbase-site.xml and restart your Master:
 <property>
 ----
 
+=== Migrating From Old Implementation
+The coprocessor `org.apache.hadoop.hbase.rsgroup.RSGroupAdminEndpoint` is
+deprected, but for compatible, if you want the pre 3.0.0 hbase client/shell
+to communicate with the new hbase cluster, you still need to add this
+coprocessor to master. And if this coprocessor is specified, the
+`hbase.balancer.rsgroup.enabled` flag will be set to true automatically to
+enable rs group feature.
+
+The main difference comparing to the old implementation is that, now the
```

Comment:
is there an implementation reason we don't fill in the tables for RSGroupInfo in this case?

Reply:
The old implementation is a bit broken, as we can set rsgroup for a namespace, and you can never get this information from the RSGroupInfo. In the new implementation there are two Admin methods for getting the information.

Reply:
please include that information in this paragraph.